src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -282,21 +318,19 @@ async def run(self) -> None:
 
             try:
                 pairs_data: List[InvBatPair] = self._get_components_data(
-                    request.batteries
+                    request.batteries, request.force
```

Comment:
hmm the initial proposed solution #321 was to use plain `force`, though I can change it if you feel `include_broken` will be more meaningful

Reply:
I know, and I do now in the context of the full PR and how `force` could be misinterpreted.  But it would be good to have @shsms and @mathias-baumann-frequenz confirm if they agree this makes more sense before proceeding with the change.

Reply:
The field in the request object is more important to get right now,  rather than the private method parameters.  And I agree that it could be better than `force`.  

The sdk doesn't have ultimate control over setting power.  This parameter is just to decide whether requests should be forwarded to the microgrid API,  where it can still get ignored.  The microgrid API can't expose a must-use type parameter.

Some options:

1. `request.must_forward`: suggesting the sdk can't block because it has not received enough data.
2. `request.override_health_checks`: suggesting that sdk's battery health check shouldn't block requests from reaching the microgrid API.

I like `request.must_forward` for now.

Reply:
yup,  I'll use `include_broken` there as well,  in the first draft.